include/message_filters/subscriber.h

```diff
@@ -52,22 +53,34 @@ class SubscriberBase
    *
    * If this Subscriber is already subscribed to a topic, this function will first unsubscribe.
    *
-   * \param node The rclcpp::Node::SharedPtr to use to subscribe.
+   * \param node The rclcpp::Node::SharedPtr or rclcpp_lifecycle::LifeCycleNode::SharedPtr  to use to subscribe.
```

Comment:
This is kind of inaccurate, actually, it's really more like:

```suggestion
   * \param node Shared pointer to the rclcpp::Node::SharedPtr object, or to an object
   *   that fulfills the required NodeT interface, to use to subscribe.
```

It need not be just `rclcpp::Node` or `rclcpp_lifecycle::LifeCycleNode`, it could be other things as well.